handler.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var request = require('request');
+var services = require('./services/index')
+
+module.exports.hello = (event, context, callback) => {
+  const body = JSON.parse(event.body);
+
+  let payload = services(body)
+
+  let teamURL = "https://outlook.office.com/MICROSOFT_TEAMS_CONNECTOR_URL"
```

Comment:
@sadikay can you set MICROSOFT_TEAMS_CONNECTOR_URL as environment variable or get from param as we talked today ?

Reply:
@muhammet alright 👍